db/migrate/20180423232334_create_feedbook_employee_has_sessions.rb

```diff
@@ -0,0 +1,15 @@
+class CreateFeedbookEmployeeHasSessions < ActiveRecord::Migration[5.1]
```

Comment:
Again, I'd separate this and the schema change into a previous commit so we can have atomic changes